projects/solar/test/test_mppt.c

```diff
@@ -0,0 +1,84 @@
+#include "log.h"
+#include "mppt.h"
+#include "spv1020_mppt.h"
+#include "spv1020_mppt_defs.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+#define TEST_SPI_PORT SPI_PORT_2
+
+void setup_test() {
+  gpio_init();
+  SpiSettings spi_settings = { .baudrate = 60000,
+                               .mode = SPI_MODE_3,
+                               .mosi = { .port = GPIO_PORT_B, 15 },
+                               .miso = { .port = GPIO_PORT_B, 14 },
+                               .sclk = { .port = GPIO_PORT_B, 13 },
+                               .cs = { .port = GPIO_PORT_B, 12 } };
```

Comment:
`make format` will probably reformat this - Travis did something strange with this PR and didn't run.